### ISSUE TITLE
about the partition of training and validation sets

### ISSUE THREAD

**Issue by string-ellipses** [phizaz/diffae#76]
I am confused about the division of training and validation sets, because I see "self.val_data = self.train_data ” in experiment.py (line 184), which seems to cause the training set and validation set to not be separatedd, thus affecting a fair evaluation of model performance.
<img width="637" alt="1851842d77f50ef37c30c163a505003" src="https://github.com/phizaz/diffae/assets/126571562/884a80a8-5b9a-4cd5-a232-c668b159ed93">

perhaps there are setps in other sections to separate the training and validation sets?

**Comment by phizaz** [comment]
Validation in generative models is done a bit differently. For example, FID calculation takes 50k images to get a reasonably stable result. If all of the 50k images are set aside for validation, there will be only 20k images left for training which is impractical.
